docs/source/unit-testing.md

```diff
@@ -27,41 +21,35 @@ actors\
     myactor\
         actor.py
         tests\
-            test_mytest.py
+            unit_test.py
 ```
 
 ### Naming conventions
 
 To have the tests found and carried out by
-[pytest framework](https://pytest.org), follow these rules:
-
-- All tests have to reside in the `test_*.py` or `*_test.py` files.
-- Test functions outside of the class have to be prefixed by `test_`.
-- Test methods with the `test_` prefix have to reside in the `Test` prefixed classes.
-- Tests should have unique filenames.
```

Comment:
Is it fixed? I think that there was reason for that because of our CI. (ping: @examon @vojtechsokol ??)

Reply:
To be more specific - the requirement of unique testfile name is actually a requirement of `pytest`. But that would apply only if we run all tests of all actors at once, like `pytest <leapp-repository/repos>`. But we call `pytest` per actor, like `pytest <specific actor folder>`. This way the need for unique filenames is limited to the `<actor>/tests/` folder which is fine because you can't have two files with the same name in a folder. The only corner case would be if someone creates subfolders in the `<actor>/tests/` folder and create same-name testfiles under them. That is a negligible risk though IMO.

Reply:
Thanks for explanation! :)